### PR TITLE
migrate on new shelf version

### PR DESCRIPTION
**Pull request by vgordievskiy**

### CODE REVIEW

File: lib/src/server_impl.dart [hunk 1]
```diff
@@ -548,8 +548,14 @@ _ErrorHandler _findErrorHandler(int statusCode, String path) {
     if (e.urlPattern == null) {
       return true;
     }
-    var match = e.urlPattern.firstMatch(path);
-    return match != null ? match[0] == path : false;
```
Comment (Pacane) [inline lib/src/server_impl.dart:552]: What was the problem with this?

Comment (vgordievskiy) [inline lib/src/server_impl.dart:552]: With new shelf (after 0.6.0) have problem with url matching

Comment (vgordievskiy) [inline lib/src/server_impl.dart:552]: All auto tests passed

**Comment by Pacane** [comment]
LGTM

**Comment by Pacane** [comment]
@vgordievskiy Oh, I just noticed, shelf went  0.6.3. Would you mind re-updating this, make sure all tests pass and I'll merge this?

Thanks

**Comment by vgordievskiy** [comment]
Hi @Pacane,

I have checked it - all tests passed with the shelf 0.6.3.

**Comment by Pacane** [comment]
Can you commit that please? Thanks for the contribution!

**Comment by vgordievskiy** [comment]
Update dependecy version for shelf.
Ready to merge.

**Comment by vgordievskiy** [comment]
@Pacane 
Thanks)

**Pull request merged**